src/gui/src/dbDescriptors.cpp

```diff
@@ -843,13 +869,13 @@ void DbNetDescriptor::highlight(std::any object,
 
   auto is_source_bterm = [](odb::dbBTerm* bterm) -> bool {
     const auto iotype = bterm->getIoType();
-    return iotype == odb::dbIoType::OUTPUT ||
+    return iotype == odb::dbIoType::INPUT ||
```

Comment:
Why the swap?

Reply:
These two were incorrect. This is checking if the bterm is a source (which it is as an INPUT, but not as an OUTPUT). I think when I reworked this earlier, I copied the Iterm and didn't double check.